include/math/QMod.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
In general to all math changes here: Compiler probably will do it better than you if not with -O1 then with -O3, or maybe you have test, then, please, prove me wrong